### PR TITLE
added Azure docs

### PR DESCRIPTION
**Pull request by kmcclenn**
## Summary

This PR adds instructions for setting up the Azure sandbox.

## Changes

- [x] Documentation

## Testing Done

```
npm install
npm run start
```
Changes shown here:

Screenshot 2024-08-14 at 7 58 05 PM

### CODE REVIEW

File: docs/getting_started.mdx [hunk 1]
```diff
@@ -135,6 +136,68 @@ the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080`
 running the [OpenHouse Catalog Service](./intro.md#catalog-service).
 :::
     </TabItem>
+    <TabItem  value="azure" label="Azure">
+# OpenHouse with Spark & Azure
+
+In this guide, we will set up a running sandbox environment and experiment with some simple SQL commands. Our
+environment includes the both [Catalog Service](./intro.md#catalog-service) and [House Table Service](./intro.md#house-table-service).
+It also utilizes [a Spark 3.1 engine](https://spark.apache.org/releases/spark-release-3-1-1.html) and many
```
Comment (sumedhsakdeo) [inline docs/getting_started.mdx:144]:
```suggestion
It also utilizes [a Spark 3.1 engine](https://spark.apache.org/releases/spark-release-3-1-1.html) and
```

File: docs/getting_started.mdx [hunk 2]
```diff
@@ -135,6 +136,68 @@ the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080`
...
+Azure services provisioned with [Terraform](https://developer.hashicorp.com/terraform?ajs_aid=a460286e-8e8f-4072-aa29-09a063308053&product_intent=terraform).
+In this walkthrough, we will spin up this sandbox environment and deploy the OpenHouse services. Then, we will
```
Comment (sumedhsakdeo) [inline docs/getting_started.mdx:146]:
```suggestion
In this walkthrough, we will spin up this sandbox environment and deploy the OpenHouse services in Azure. Then, we will
```

File: docs/getting_started.mdx [hunk 3]
```diff
@@ -135,6 +136,68 @@ the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080`
...
+message.
+
+```shell
+openhouse$main>  ./gradlew build
+```
+
+### Step 2
+Setup an Azure account and create/choose a subscription. Run `az login` to login to your Azure account and subscription from the terminal.
+
+### Step 3
```
Comment (sumedhsakdeo) [inline docs/getting_started.mdx:172]: Install terraform?

Comment (kmcclenn) [reply docs/getting_started.mdx:172]: That is in the prerequisites section

File: docs/getting_started.mdx [hunk 4]
```diff
@@ -135,6 +136,68 @@ the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080`
...
+Navigate into the container environment, and initialize and apply the Terraform configuration:
+```shell
+cd infra/recipes/terraform/azure/environments/container
```
Comment (sumedhsakdeo) [inline docs/getting_started.mdx:175]: Did we not have to create tfvars files?

Comment (kmcclenn) [reply docs/getting_started.mdx:175]: No - when talking with @HotSushi  and @jiang95-dev  we changed the default backend to local storage and then added info in the README for remote backend configuration. So default is not changing the backend.

File: docs/getting_started.mdx [hunk 5]
```diff
@@ -135,6 +136,68 @@ the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080`
...
+terraform init
+terraform apply
+```
+
+### Step 4
+Navigate into the sandbox environment, and initialize and apply the Terraform configuration:
+```shell
+cd ../sandbox
```
[the remark this answers is not in the document]
Comment (kmcclenn) [reply docs/getting_started.mdx]: Done

File: docs/getting_started.mdx (hunk 4 above)
Comment (sumedhsakdeo) [reply docs/getting_started.mdx:175]: I see same for step-4?

Comment (kmcclenn) [reply docs/getting_started.mdx:175]: yes same for both environments

**Review by sumedhsakdeo — APPROVED** [review]

**Pull request merged**